css/css-animations/set-animation-play-state-to-paused-001.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <meta charset="utf-8">
+    <title>Dynamically set animation-play-state to paused</title>
+    <link rel="author" title="Igalia S.L." href="https://www.igalia.com/">
+    <link rel="help" href="https://drafts.csswg.org/css-animations-1/#animation-play-state">
+    <meta name="flags" content="animated">
```

Comment:
@birtles I had copied this "animated" meta flag from other tests for CSS animations. However, I'm not exactly sure about the purpose of this flag... It seems the manual tests changed in #11215 were exactly those having the meta flag. Do you know more about this?

Reply:
OK I removed this flag, per #5293